SparkMAXDriveTrain/src/main/java/org/usfirst/frc2130/SparkMAXDriveTrain/subsystems/DriveTrainSubsystem.java

```diff
@@ -39,6 +57,27 @@ public DriveTrainSubsystem() {
         // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTRUCTORS
 
         // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTRUCTORS
+
+        //Here we are giving definitions to our previously named variables the format is
+        //we first call the name rightMotorMaster, then say new, what it is, CANSparkMax, then we state the CAN ID and what type
+        //of motor it is -Cory
+        rightMotorMaster = new CANSparkMax(1, MotorType.kBrushless);
+        rightMotorSlave = new CANSparkMax(2, MotorType.kBrushless);
+        
+        leftMotorMaster = new CANSparkMax(3, MotorType.kBrushless);
+        leftMotorSlave = new CANSparkMax(4, MotorType.kBrushless);
+        
+        diffydrivey = new DifferentialDrive(leftMotorMaster, rightMotorMaster);
+
+        //sorry cheif I honestly don't know -Cory
+        diffydrivey.setSafetyEnabled(true);
+        diffydrivey.setExpiration(0.1);
+        diffydrivey.setMaxOutput(1.0);
+
+
+        //Here we are setting the slaves to make it follow the master so they mirror what the master does -Cory
+        leftMotorSlave.follow(leftMotorMaster);
+        rightMotorSlave.follow(rightMotorMaster);
```

Comment:
This would be better as a method that you call because you will need to call these 2 lines of code any time you have to use these motors.